assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -265,15 +276,33 @@ function MediaPane(props) {
               options={FILTERS}
               placement={Placement.BOTTOM_START}
             />
-            <Primary onClick={openMediaPicker}>
-              {__('Upload', 'web-stories')}
-            </Primary>
+            {isSearching && Boolean(media.length) && (
```

Comment:
Nit: `Boolean()` cast isn't necessary.

Reply:
Done

Reply:
What if length is 0?

Don‘t want to render 0 accidentally

https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
I will just do `media.length > 0`, I think this make more sense.